testing/py-pygfm/APKBUILD

```diff
@@ -1,36 +1,36 @@
 # Contributor: Leonardo Arena <rnalrd@alpinelinux.org>
-# Maintainer:
+# Contributor: Stefan Wagner <stw@bit-strickerei.de>
+# Maintainer: Stefan Wagner <stw@bit-strickerei.de>
 pkgname=py-pygfm
-pkgver=0.1.3
+_pkgname=py-gfm
+pkgver=0.1.4
 pkgrel=0
-pkgdesc="Basic immutable container types for Python"
-url="https://github.com/tomchristie/itypes"
+pkgdesc="Github-Flavored Markdown for Python-Markdown"
+url="https://github.com/zopieux/py-gfm"
 arch="noarch"
 license="BSD-3-Clause"
-depends=""
 makedepends="python2-dev python3-dev py-setuptools"
 checkdepends="py-codecov py-coverage py-coveralls py-markdown"
 subpackages="py2-${pkgname#py-}:_py2 py3-${pkgname#py-}:_py3"
-source="$pkgname-$pkgver.tar.gz::https://github.com/Zopieux/py-gfm/archive/$pkgver.tar.gz"
-builddir="$srcdir"/py-gfm-$pkgver
+source="$pkgname-$pkgver.tar.gz::https://github.com/Zopieux/$_pkgname/archive/$pkgver.tar.gz"
```

Comment:
please revert this change back - it is very useful to navigate to repo from url without substitution